munit/js/src/main/scala/java/nio/file/Files.scala

```diff
@@ -24,4 +31,59 @@ object Files {
     }
     result
   }
+
+  def createDirectories(dir: Path): Path = JSIO.inNode {
+    JSFs.mkdirSync(dir.toString(), js.Dynamic.literal(recursive = true))
+    dir
+  }
+
+  def createTempDirectory(prefix: String): Path = JSIO.inNode {
+    Paths.get(JSFs.mkdtempSync(JSPath.join(JSOS.tmpdir(), prefix)))
+  }
+
+  def exists(path: Path): Boolean = JSIO.inNode {
+    JSIO.exists(path.toString)
+  }
+
+  def deleteIfExists(path: Path): Boolean = JSIO.inNode {
+    if (JSIO.exists(path.toString)) {
+      JSFs.unlinkSync(path.toString)
+      true
+    } else {
+      false
+    }
+  }
+
+  def write(path: Path, bytes: Array[Byte], options: OpenOption*): Path =
```

Comment:
The JDK boot classpath takes precedence over regular classpath entries during compilation, while regular classpath entries are used during linking time (since they have sjsir files). Therefore, varargs need to use the same erasure signature as Java varargs
```suggestion
  def write(path: Path, bytes: Array[Byte], options: Array[OpenOption]): Path =
```

Reply:
👍 makes sense (although is it expected that this works in tests?)

Reply:
No need to change it if it works in the tests